pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -193,7 +195,7 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 			fmt.Errorf("failed to predicates, node info for %s not found", node.Name)
 		}
 
-		if node.Allocatable.MaxTaskNum <= len(nodeInfo.Pods()) {
```

Comment:
why not use Pod() as before.

Reply:
The struct nodeInfo is a map and the key is Pods